demisto_sdk/commands/validate/validators/BC_validators/BC116_is_breaking_agentix_action_output_backwards.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Set
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.agentix_action import AgentixAction
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = AgentixAction
+
+
+class IsBreakingAgentixActionOutputBackwardsValidator(BaseValidator[ContentTypes]):
+    error_code = "BC116"
+    description = "Validate that no context output keys were removed from the Agentix action's output section."
+    rationale = "To ensure we don't break backward compatibility."
+    error_message = "The following output keys: {0} have been removed, please undo."
+    related_field = "outputs"
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.RENAMED, GitStatuses.MODIFIED]
```

Comment:
is renamed status relevant here?

Reply:
`BC101` and `BC102` also contain renamed.
If a file has both been changed **and** renamed, it is defined as **renamed**, perhaps this is the reason.